pkg/controller/migmigration/dvm.go

```diff
@@ -116,6 +118,21 @@ func (t *Task) hasDirectVolumeMigrationCompleted(dvm *migapi.DirectVolumeMigrati
 	return completed, failureReasons, progress
 }
 
+func (t *Task) getDVMProgress(runningPods []*migapi.RunningPod) []string {
+	progress := []string{}
+	for _, pod := range runningPods {
+		p := fmt.Sprintf("rsync client pod %s is running", path.Join(pod.Namespace, pod.Name))
```

Comment:
```suggestion
		p := fmt.Sprintf("Pod %s/%s: Running", pod.Namespace, pod.Name)
		if pod.LastObservedProgressPercent != "" {
		        p = fmt.Sprintf("Pod %s/%s: Running, %s completed", pod.LastObservedProgressPercent)
		}
		progress = append(progress, p)
```

Reply:
I don't think we absolutely need to show the transfer rate as of now. The UI doesnt have anything for the transfer rate. 

Apart from these, it would be nice if we could show duration at the end of the string, just like Velero Backup / Restore. But again, not a hard requirement.

Reply:
What happens when a Pod fails / succeds? This progress info is only for in-progress pods.

Reply:
@pranavgaikwad would it be hard to add something in the UI to show the transfer rate? I guess why can we not have the transfer rate in CLI and ignore it in the UI? Feels weird if we have to remove something from the CR just because of UI